Build failure on Linux, looks like `LOOKUPTABLE_SIZE` will only be available on Windows (`#ifdef HAS_DX`):

```
In file included from GUIFont.cpp:22:0:
GUIFontTTF.h:162:26: error: ‘LOOKUPTABLE_SIZE’ was not declared in this scope
   Character *m_charquick[LOOKUPTABLE_SIZE];     // ascii chars (7 styles) here
                          ^
make[1]: *** [GUIFont.o] Error 1
```

--- a/xbmc/guilib/GUIFontTTF.cpp
+++ b/xbmc/guilib/GUIFontTTF.cpp
@@ -615,7 +615,7 @@ CGUIFontTTFBase::Character* CGUIFontTTFBase::GetCharacter(character_t chr)
   if (letter < 255)
   {
     character_t ch = (style << 8) | letter;
-    if (m_charquick[ch])
+    if (ch < LOOKUPTABLE_SIZE && m_charquick[ch])
       return m_charquick[ch];
   }
 

--- a/xbmc/guilib/GUIFontTTF.h
+++ b/xbmc/guilib/GUIFontTTF.h
@@ -42,6 +42,7 @@ using namespace DirectX;
 using namespace DirectX::PackedVector;
 #endif
 
+constexpr size_t LOOKUPTABLE_SIZE = 256 * 8;
 // forward definition
 class CBaseTexture;
 
@@ -157,7 +158,7 @@ protected:
   color_t m_color;
 
   Character *m_char;                 // our characters
-  Character *m_charquick[256*7];     // ascii chars (7 styles) here
+  Character *m_charquick[LOOKUPTABLE_SIZE];     // ascii chars (7 styles) here
   int m_maxChars;                    // size of character array (can be incremented)
   int m_numChars;                    // the current number of cached characters
 

